woocommerce/woocommerce
# Add Pricing item to the Quick Actions menu per Variation item

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #39788



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure feature `New product editor` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure feature `product-variation-management` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
3. Go to `/wp-admin/admin.php?page=wc-admin&path=/add-product&tab=variations` and click `Add variation options` button from the `Variations` tab
4. Add at least one Variation attribute and at least one value. Then click `Add` button from the `Add variation options` modal.
5. Wait until the variations are generated
6. A list a variation should be shown depending on the options selected
7. In the Variations table, each row should have a dropdown icon button at the right most place
8. When clicking a dropdown menu should be shown
9. From the menu a `Pricing` item should be shown
10. When clicking the `Pricing` a sub menu should be shown 
<img width="727" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/5245be67-a340-49da-9fe3-1f1494fbaf28">

11. `Set list price` should let you change the regular price of the variation by the amount you enter in the window prompt
12. `Increase list price` and `Decrease list price` should let you change the regular price of the variation in a fixed amount or using a percentage if you add `%` at the end of the value in the prompt
13. `Set sale price` item is the same as point 11 but it changes the sale price of the variation
14. `Increase sale price` and `Decrease sale price` items are the same as point 12 but they affect the sale price of the variation
15. `Schedule sale` should let you change the `Start date of sale price` of the variation
16. To check if those changes are working well consider previewing the specific variation you are modifying.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
